player/lua/osc.lua

```diff
@@ -2241,8 +2240,8 @@ function hide_osc()
         -- no-op and won't render again to remove the osc, so do that manually.
         state.osc_visible = false
         render_wipe()
-    elseif (user_opts.fadeduration > 0) then
-        if not(state.osc_visible == false) then
+    elseif user_opts.fadeduration > 0 then
+        if state.osc_visible then
```

Comment:
This is not strictly the same condition. The original condition would trigger if `state.osc_visible = nil`, but the new one will not.

Reply:
That is true, but `osc_visible` is only ever true or false.

Reply:
Perhaps, but if this PR is supposed to be a purely for code style changes only, it might be worth guaranteeing there are no changes in the code logic, even if theoretically the behaviour will be identical. A simple `if state.osc_visible ~= false` would be a simple change to ensure this guarantee.

Reply:
I mean I can do that if it's really insisted upon, but `if state.osc_visible then` does already exist in several places in the code already so I could argue that I'm just matching that style (even if the logic is technically different).

Reply:
I don't really have strong feelings about it, I just thought I should point it out since it seems to be the only change in the PR that isn't purely stylistic. Maybe wait and see what one of the maintainers think.